modules/salt/pages/yomi.adoc

```diff
@@ -0,0 +1,174 @@
+[[yomi.installer]]
+= Install with Yomi
+
+Yomi (yet one more installer) is an installer for openSUSE operating systems.
+It is a SaltStack state, and can be used for installing Salt clients.
+
+Yomi consists of two components:
+
+* The Yomi formula, which contains the Salt states and modules required to drive the installation.
+For the formula, see https://build.opensuse.org/package/show/systemsmanagement:yomi/yomi-formula.
+* The LiveCD operating system image, which includes the `pre-configured ``salt-minion`` service.
+For the image, see https://build.opensuse.org/package/show/systemsmanagement:yomi/openSUSE-Tumbleweed-Yomi
+
+
+To use Yomi for installing your client operating system, you need to follow this process:
+
+* Install and configure the ``salt-master``service.
+* Install the Yomi formula package.
+* Prepare the Salt pillar for the new installation.
+* Boot the new systems with the operating system image, or use PXE boot.
+
+For more information about using Yomi, see https://github.com/openSUSE/yomi/blob/master/README.md
+
+
+
+== Install and Configure salt-master
```

Comment:
In the general case this is required, but because Uyuni already provides a salt-master, this full step is not necessary.

The yomi-formula package takes care also of adding a configuration file snippet to adjust the salt-master configuration.